### DISCUSSION TITLE
Ending simulation based on a post processor value

### DISCUSSION THREAD

**Discussion by TheGreatCid** [idaholab/moose#29033] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hi all,

I am sure that I am missing an obvious solution here but it is eluding me. Is there a way to end a simulating using a post processor?

For example I want to end a simulation if the minimum element quality is too low. I have a post proc that outputs min quality, but I can't find a way to have that value interact with the executioner. 

Thanks!

**Comment by GiudGiud** [comment]
Hello

Yes, this capability just got added to MOOSE. You ll need to install the "next" branch, because it has not been released to the main branch yet.
You ll want to specify a PostprcoessorConvergence and specify nonlinear_convergence_name to the name of that object in the Executioner


Guillaume

**Reply by TheGreatCid** [reply]
Great, thanks!

**Reply by TheGreatCid** [reply]
Actually, does this method then override the normal convergence parameters? 
I want to keep non linear tolerances, but just kill the simulation when a certain mesh quality is reached.

**Reply by GiudGiud** [reply, marked as answer]
ok just use this instead with PPs on the quality
https://mooseframework.inl.gov/source/userobjects/Terminator.html

this CV object method will eventually do what you want but not yet. We dont have a "Combo" convergence to specify more than 1 criterion

**Reply by TheGreatCid** [reply]
Ah this is exactly what I was looking for! Thanks!
